CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Open Development & Community Driven
+
+This project is open-source under the [MIT license](LICENSE). All the work done is available on GitHub.
+
+## Code of Conduct
+
+This project and everyone participating in it are governed by a [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please read the [full text](CODE_OF_CONDUCT.md) so that you can read which actions may or may not be tolerated.
+
+## Bugs
+
+We are using [GitHub Issues](https://github.com/Lith/strapi-provider-upload-google-cloud-storage/issues) to manage our public bugs. We keep a close eye on this so before filing a new issue, try to make sure the problem does not already exist.
+
+---
+
+## Pull Request Process
+
+The team will review your pull request and will either merge it, request changes to it, or close it.
+
+**Before submitting your pull request** make sure the following requirements are fulfilled:
+
+1. Fork the repository and create your branch from `master`.
+    - Run `yarn install` in the repository root.
```

Comment:
It's possible using `npm` too, but... OK

Reply:
I've reproduce Strapi recommendation and we use `yarn.lock` files for this project.
Using `npm` could generate warning.